docs/api/permissions.rst

```diff
@@ -0,0 +1,188 @@
+.. _api-permissions:
+
+Permissions
+###########
+
+
+Terminology
+===========
+
+.. glossary::
+
+    Object
+        Anything that can be interacted with. Collections, records, buckets, groups
+        are all objects.
+
+    Principal
+        An entity that can be authenticated. Principals can be individual people,
+        applications, services, or any group of such things.
+
+    Group
+        A group associates a name to a list of principals.
+
+    Permission
+    Permissions
+        A permission is an action that can be performed on an object.
+        Examples of permissions are «read», «write», or «create».
+
+    ACE
+    Access Control Entity
+        An ACE associates a permission to objects and principals, and allows
+        to describe rules like «*Members of group admins can create collections*».
+        Using #a pseudo-code syntax: ``collections:create = ['group:admins',]``.
+
+    ACL
+        A list of ACEs
+
+Objects
+=======
+
+Any set of objects defined in *Kinto* can be given a number of permissions.
```

Comment:
(imho) This sentence is out of context - consider moving it to the end of this section, or removing it entirely.